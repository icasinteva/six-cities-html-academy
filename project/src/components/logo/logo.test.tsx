import { render } from '@testing-library/react';
import { createMemoryHistory } from 'history';
import HistoryRouter from '../history-route';
import Logo from './logo';

describe('Component: Logo', () => {
  test('should render correctly', () => {
    const history = createMemoryHistory();
    const { container } = render(
      <HistoryRouter history={history}>
        <Logo className='header' width='81' height='41' />
      </HistoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
