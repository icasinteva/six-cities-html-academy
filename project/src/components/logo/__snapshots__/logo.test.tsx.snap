// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Logo should render correctly 1`] = `
<div>
  <a
    class="header__logo"
    href="/"
  >
    <img
      alt="6 cities logo"
      class="header__logo-link"
      height="41"
      src="img/logo.svg"
      width="81"
    />
  </a>
</div>
`;
